src/reducers/job.js

```diff
@@ -29,6 +29,14 @@ export default {
         immutable(state, {
           jobStatus: { $set: STATUS.SUCCESS },
         }),
+      [ActionTypes.EXPORT_SNAPSHOT_JOB]: (state, action) =>
+        immutable(state, {
+          jobId: { $set: action.payload.jobId },
+        }),
+      [ActionTypes.EXPORT_SNAPSHOT_SUCCESS]: (state) =>
+        immutable(state, {
+          jobStatus: { $set: STATUS.SUCCESS },
```

Comment:
You may need a handler for failure.

Reply:
What are you talking about? Our API never fails!

Reply:
ohhh that's probably what's missing on snapshot create failure too, I was following the same action pattern - https://broadworkbench.atlassian.net/browse/DR-2181

Reply:
I think I found the actual issue for DR-2181 (b/c I'm encountering the same issue for export) -- We weren't handling the case that the poll job worker had an exception. We were just showing the error message. I've changed it so that we're also handling the exception corresponding to either create snapshot or export snapshot.